app/lib/coursesHelper.coffee

```diff
@@ -178,7 +178,7 @@ module.exports =
             numStarted: 0
             # numCompleted: 0
           }
-          isOptional = level.get('practice') or level.get('assessment')
+          isOptional = level.get('practice') or level.get('assessment') or level.isLadder()
```

Comment:
This change filters out the `isLadder` levels on the teacher side which has parity with the student side as well as our decision that arenas are not required.